docs/developing-for-pulsar/publishing.md

```diff
@@ -3,69 +3,84 @@ title: Publishing
 layout: doc.ejs
 ---
 
-Pulsar bundles a command line utility called `ppm` into every installation of Pulsar, to search for and install packages via the command line. This is invoked by using the `pulsar` command with the `-p` or `--package` option. Optionally, you are still able to call PPM directly via `ppm` in the command line. The `pulsar -p` command can also be used to publish Pulsar packages to the public registry and update them.
+Pulsar bundles a command line utility called `ppm` into every installation of
+Pulsar, to search for and install packages via the command line. This is invoked
+by using the `pulsar` command with the `-p` or `--package` option. The `pulsar -p`
+command can also be used to publish Pulsar packages to the public registry and
+update them.
```

Comment:
Two issues:

1. Please keep the soft-wrap rather than hard line breaks;
2. The sentence you removed is still valid for most users, so it should be retained.

Reply:
1. ok, not my backyard so I will follow the standards here, in 90% of the projects I'm following we're using hard wrapping, because, you know, easier to diff, but no strong opinions.
2. if you are referring to `Optionally, you are still able to call PPM directly via `ppm` in the command line`, to me it was kinda implicit, but I think a little rephrase could help out understanding.